openshift_tools/python-openshift-tools.spec

```diff
@@ -38,27 +38,105 @@ cp -p web/*.py %{buildroot}%{python_sitelib}/openshift_tools/web
 # openshift_tools files
 %files
 %dir %{python_sitelib}/openshift_tools
+%dir %{python_sitelib}/openshift_tools/monitoring
+%{python_sitelib}/openshift_tools/monitoring/__init*
 %{python_sitelib}/openshift_tools/*.py
 %{python_sitelib}/openshift_tools/*.py[co]
 
 
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-pcp subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-pcp
+Summary:       OpenShift Tools Monitoring Python Package
+#Requires:      python2,python-openshift-tools,python-zbxsend,python-openshift-tools-web,python-pcp,atomic-openshift,python-awscli
+Requires:      python2,python-openshift-tools,python-pcp
+BuildArch:     noarch
+
+%description monitoring-pcp
+Tools developed for monitoring OpenShift.
 
+%files monitoring-pcp
+#%dir %{python_sitelib}/openshift_tools/monitoring
+%{python_sitelib}/openshift_tools/monitoring/pminfo*.py
+%{python_sitelib}/openshift_tools/monitoring/pminfo*.py[co]
 
 # ----------------------------------------------------------------------------------
-# python-openshift-tools-monitoring subpackage
+# python-openshift-tools-monitoring-docker subpackage
 # ----------------------------------------------------------------------------------
-%package monitoring
+%package monitoring-docker
 Summary:       OpenShift Tools Monitoring Python Package
-Requires:      python2,python-openshift-tools,python-zbxsend,python-openshift-tools-web,python-pcp,atomic-openshift,python-awscli
+#Requires:      python2,python-openshift-tools,python-zbxsend,python-openshift-tools-web,python-pcp,atomic-openshift,python-awscli
+Requires:      python2,python-openshift-tools,python-pcp
 BuildArch:     noarch
 
-%description monitoring
+%description monitoring-docker
 Tools developed for monitoring OpenShift.
 
-%files monitoring
-%dir %{python_sitelib}/openshift_tools/monitoring
-%{python_sitelib}/openshift_tools/monitoring/*.py
-%{python_sitelib}/openshift_tools/monitoring/*.py[co]
+%files monitoring-docker
+%{python_sitelib}/openshift_tools/monitoring/dockerutil.py
+%{python_sitelib}/openshift_tools/monitoring/dockerutil.py[co]
+
+
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-zagg subpackage
```

Comment:
I thought we talked about having both zagg-client and zagg-server sub packages.

In this model, if I just want zagg_client.py, I have to install all of zagg server, and all of it's deps.

Isn't that what we're trying to avoid on Tower?